addons/special/CfgWeapons.hpp

```diff
@@ -5,6 +5,35 @@ class CfgWeapons {
     class Uniform_Base;
     class UniformItem;
     class U_I_C_Soldier_Bandit_2_F;
+    class HeadgearItem;
+    class ItemCore;
+
+    // Jack
+    class CUP_H_C_Beret_01: ItemCore {
+        class ItemInfo: HeadgearItem {
+            class HitpointsProtectionInfo {
+                class Head;
+            };
+        };
+    };
+
+    class CLASS(Jack_Beret): CUP_H_C_Beret_01 {
+        author = "GilleeDoo";
+        scope = 2;
+        displayName = "Beret, True Freedom Edition";
+        descriptionShort = "It has armour, somehow.";
```

Comment:
Can't we just say it has a metal plate inside it? I am not a fan of those "magical" things.

Reply:
It is lined with a composite shell made with best in class Ultra High Molecular Weight Polyethylene (UHMWPE) material.